app/src/organisms/DropTipWizard/index.tsx

```diff
@@ -252,22 +238,19 @@ export const DropTipWizardComponent = (
   }
   const handleJog: Jog = (axis, dir, step) => {
     if (createdMaintenanceRunId != null) {
-      chainRunCommands(
-        createdMaintenanceRunId,
-        [
-          {
-            commandType: 'moveRelative',
-            params: {
-              pipetteId: MANAGED_PIPETTE_ID,
-              distance: step * dir,
-              axis,
-            },
-          },
-        ],
-        true
-      )
+      createRunCommand({
+        maintenanceRunId: createdMaintenanceRunId,
+        command: {
+          commandType: 'moveRelative',
+          params: { pipetteId: MANAGED_PIPETTE_ID, distance: step * dir, axis },
+        },
+        waitUntilComplete: true,
+        timeout: JOG_COMMAND_TIMEOUT_MS,
+      })
         .then(data => {})
-        .catch((e: Error) => {})
+        .catch((e: Error) =>
+          setErrorMessage(`error issuing jog command: ${e.message}`)
```

Comment:
```suggestion
          setErrorMessage(`Error issuing jog command: ${e.message}`)
```
Definitely a nit, but it seems to be consistent with how we do error messages.